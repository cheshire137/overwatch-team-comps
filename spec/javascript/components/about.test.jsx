import renderer from 'react-test-renderer'

import About from '../../../app/assets/javascripts/components/about.jsx'

describe('About', () => {
  let component = null

  beforeEach(() => {
    component = <About />
  })

  test('matches snapshot', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
