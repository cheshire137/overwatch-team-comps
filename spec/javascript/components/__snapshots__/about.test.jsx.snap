exports[`About matches snapshot 1`] = `
<div
  className="container about-page">
  <h1>
    About
  </h1>
  <p>
    Thanks to Blizzard for Overwatch. 
    <i
      aria-hidden="true"
      className="fa fa-heart" />
     This app was built with 
    <a
      href="https://facebook.github.io/react/"
      rel="noopener noreferrer"
      target="_blank">
      React
    </a>
     and 
    <a
      href="http://rubyonrails.org/"
      rel="noopener noreferrer"
      target="_blank">
      Ruby on Rails
    </a>
    .
  </p>
</div>
`;
